prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MO/MO-receiver-transform.yml

```diff
@@ -0,0 +1,40 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
```

Comment:
You can add the following constants and use them later in this transform.

constants:
  PID: '/PATIENT_RESULT/PATIENT/PID'
  ORC: '/PATIENT_RESULT/ORDER_OBSERVATION/ORC'
  OBR: '/PATIENT_RESULT/ORDER_OBSERVATION/OBR'

Reply:
Only one of those is applicable, but I can add it.